dirsrvtests/tests/suites/indexes/regression_test.py

```diff
@@ -260,6 +301,108 @@ def test_reject_virtual_attr_for_indexing(topo):
+        1. Start a Reindex task on 'cn' and wait until it is completed
+        2. Check that task has a status
+        3. Check that exit code is 0
+        4. Start a Reindex task on 'badattr' and wait until it is completed
+        5. Check that task has a status
+        6. Check that exit code is 0
+        1. Success
+        2. Success
+        3. Success
+        4. Success
+        5. Success
+        6. Success
+    """
+
+    tasks = Tasks(topo.standalone)
+    # completed reindex tasks MUST have a status because freeipa check it.
+
+    # Reindex 'cn'
+    tasks.reindex(
+        suffix=DEFAULT_SUFFIX,
+        attrname='cn',
+        args={TASK_WAIT: True}
+    )
+    reindex_task = Task(topo.standalone, tasks.dn)
+    assert reindex_task.status()
+    assert reindex_task.get_exit_code() == 0
+
+    # Reindex 'badattr'
+    tasks.reindex(
+        suffix=DEFAULT_SUFFIX,
+        attrname='badattr',
+        args={TASK_WAIT: True}
+    )
+    reindex_task = Task(topo.standalone, tasks.dn)
+    assert reindex_task.status()
+    # Bad attribute are skipped without setting error code
+    assert reindex_task.get_exit_code() == 0
+
+
+def test_task_and_be(topo, add_backend_and_ldif_50K_users):
+    """Check that backend is writable after finishing a tasks
+
+    :id: 047869da-7a4d-11ee-895c-482ae39447e5
+    :setup: Standalone instance + a second backend with 50K users
+    :steps:
+        1. Start an Import task and wait until it is completed
+        2. Modify the suffix entry description
+        3. Start a Reindex task on all attributes and wait until it is completed
+        4. Modify the suffix entry description
+        5. Start an Export task and wait until it is completed
+        6. Modify the suffix entry description
+        1. Success
+        2. Success
+        3. Success
+        4. Success
+        5. Success
+        6. Success
+    """
+
+    tasks = Tasks(topo.standalone)
+    user = UserAccount(topo.standalone, f'uid=user00001,ou=people,{SUFFIX2}')
+    ldif_file = add_backend_and_ldif_50K_users
+
+    # Import
+    tasks.importLDIF(
+        suffix=SUFFIX2,
+        input_file=ldif_file,
+        args={TASK_WAIT: True}
+    ) == 0
+    user.set('description', 'test_task_and_be tc1')
+
+    # Reindex some attributes
+    assert tasks.reindex(
+        suffix=SUFFIX2,
+        attrname=[ 'description', 'rdn', 'uid', 'cn', 'sn', 'badattr' ],
+        args={TASK_WAIT: True}
+    ) == 0
+    user.set('description', 'test_task_and_be tc2')
+    users = UserAccounts(topo.standalone, SUFFIX2, rdn=None)
+    user = users.create(properties={
+        'uid': 'user1',
+        'sn': 'user1',
+        'cn': 'user1',
+        'uidNumber': '1001',
+        'gidNumber': '1001',
+        'homeDirectory': '/home/user1'
+    })
+
+    # Export
+    assert tasks.exportLDIF(
+        suffix=SUFFIX2,
+        output_file=f'{ldif_file}2',
+        args={TASK_WAIT: True}
+    ) == 0
+    user.set('description', 'test_task_and_be tc3')
```

Comment:
Just to be on a safe side, I think it makes sense to assert that the user has the description in the end.

Reply:
Fixed.